### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by IOAyman**
- [x] Updated `dotenv`
- [x] Handle missing `.env` file
- [x] Handle missing CLIENT_ID and CLIENT_SECRET env vars 
- [x] Added .editorconfig
- [x] Fixed syntax highlighting in README
- [x] Updated contributors list
- [x] Corrected license name in `package.json`
- [ ] Add Docker image definition

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
 'use strict';
 const request = require('request');
-const dotenv = require('dotenv');
+const { config } = require('dotenv');
 const _ = require('lodash');
 
 // load environment file
-dotenv.load();
+const { error: err } = config();
```
Comment (wildan3105) [inline lib/controller.js]: Thanks! I never thought about missing `.env` file. Really helpful :+1:

**Comment by wildan3105** [comment]
Thanks for the minor patches @IOAyman ! Good work :muscle:
If you find any security/bug/anything, feel free to open up an issue

**Comment by IOAyman** [comment]
@wildan3105, You're welcome :) Glad to PR anytime.

I'll add a Dockerfile before you merge

**Comment by IOAyman** [comment]
Oh, already merge :D  Will open up another PR for that in a few.

**Comment by wildan3105** [comment]
Ups sorry :smile:. I do not have any experience with Docker, so if you willing to add it, just open up and issue and we could discuss about Docker on that issue. Wait for the next your contribution! :muscle:

**Comment by wildan3105** [comment]
Sorry @IOAyman, it looked like the change you made at `controller.js` throw an error like this : 

error

So I resolved it by [commit](https://github.com/wildan3105/github-langs/commit/2373e1ccee4024ceaa53da2ec16cad4128227e1f)

**Comment by IOAyman** [comment]
@wildan3105, Actually, my commit works just fine.  
That error is cause because you did not `npm i` after merging the new code. If you pay attention to my commit (f8342e20e2fb0985dc1f4e198ec247fecf9fa28f) you would see that the `dotenv` version has been upgraded 😉

**Comment by IOAyman** [comment]
Please check again.

**Comment by wildan3105** [comment]
Oh, sorry I missed that one. I will try it later! Thanks @IOAyman

**Comment by IOAyman** [comment]
@wildan3105, Sure. TYT 😄

**Comment by wildan3105** [comment]
#25 @IOAyman If you'd like to

**Pull request merged**